README.md

```diff
@@ -31,6 +32,23 @@ to tokenize the text and search for *actual* instances of those keywords.
 Believe it or not, some website search engines lead to results that don't 
 actually contain those keywords!
 
+## Prerequisites to install
+
+General software (please search for your operating system for instructions):
+* Chromedriver 
+
+Python packages (these all can be installed via `pip`, and probably other alternatives): 
+* Selenium
+* argparse
+* numpy
+* pandas
+* spacy
+* pytesseract
+* PIL 
+* wand.image 
```

Comment:
argparse and PIL are part of the std library.

in `setup.cfg` I defined the following requirements by guess/checking my way to getting things to run

```
install_requires =
                numpy
                pandas
                spacy
                pdfminer.six
                pytesseract 
                wand 
                selenium
                chromedriver
                webdriver-manager
```

Reply:
`pip install (-e) .` will collect and install all of these.
I would mention something about the spacy extension instructions / that it auto-downloads if missing.

Reply:
cool, on it like a bonnet. Will be making a commit with these changes shortly.